### ISSUE TITLE
Hello World feedback after submission of further exercises

### ISSUE THREAD

**Issue by tejasbubane** [exercism/rikki#25]
I just received `hello-world` feedback after I had submitted next few exercises.

This is 16 hrs ago as of the creation of this issue (Monday, 5 December 2016, 8:55 UTC)

![screen shot 2016-12-05 at 2 16 29 pm](https://cloud.githubusercontent.com/assets/980783/20878574/b316adc2-baf5-11e6-83aa-5f279b1b87c2.png)

While I had submitted few more exercises before 
![screen shot 2016-12-05 at 2 16 53 pm](https://cloud.githubusercontent.com/assets/980783/20878586/cd1135b2-baf5-11e6-896f-88c5945765f6.png)

Was this due to some late processing of jobs? I remember getting instant feedback a while ago. 

Either ways, while giving `hello-world` feedback we should check if further exercises are submitted. This might also happen in case someone skips `hello-world` but then submits it sometime later.

**Comment by kotp** [comment]
re: #22

**Comment by kytrinyx** [comment]
Yeah, this is my bad. I screwed up the redis database, and ended up having to copy a backlog into a new database.
